egs/tedlium/s5_r2/local/ted_train_lm.sh

```diff
@@ -36,6 +37,18 @@ export PATH=$KALDI_ROOT/tools/pocolm/scripts:$PATH
 
 num_dev_sentences=10000
 
+bypass_metaparam_optim_opt=
+# If you want to bypass the metaparameter optimization steps with specific metaparameters
+# un-comment the following line, and change the numbers to some appropriate values.
+# You can find the values from output log of train_lm.py.
+# These example numbers of metaparameters is for 4-gram model (with min-counts)
+# running with train_lm.py.
+# The dev perplexity should be close to the non-bypassed model.
+#bypass_metaparam_optim_opt="--bypass-metaparameter-optimization=0.794,0.022,0.761,0.083,0.043,0.013,1.000,0.359,0.123,0.078,1.000,0.613,0.227,0.227"
```

Comment:
You don't need to update the non-min-count perplexities.

On Sun, Aug 14, 2016 at 9:21 PM, Ke Li <notifications@github.com> wrote:

> In egs/tedlium/s5_r2/local/ted_train_lm.sh
> <https://github.com/kaldi-asr/kaldi/pull/964#discussion_r74719386>:
>
> > @@ -36,6 +37,18 @@ export PATH=$KALDI_ROOT/tools/pocolm/scripts:$PATH
> >
> >  num_dev_sentences=10000
> >
> > +bypass_metaparam_optim_opt=
> > +# If you want to bypass the metaparameter optimization steps with specific metaparameters
> > +# un-comment the following line, and change the numbers to some appropriate values.
> > +# You can find the values from output log of train_lm.py.
> > +# These example numbers of metaparameters is for 4-gram model (with min-counts)
> > +# running with train_lm.py.
> > +# The dev perplexity should be close to the non-bypassed model.
> > +#bypass_metaparam_optim_opt="--bypass-metaparameter-optimization=0.794,0.022,0.761,0.083,0.043,0.013,1.000,0.359,0.123,0.078,1.000,0.613,0.227,0.227"
>
> It's very slow to run the script (~3-4h for generating a unpruned model)
> without using min-count. Do I need to update ppls without min-count later?
> Ke
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/964/files/f37d8f73b65a7ab16a3d862e0f91175efe5ac79a#r74719386>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVux6hekzPRCFWDnbMLUwySxVlkMsVks5qf-lFgaJpZM4JkFeX>
> .
>